docs/topics/java-read-scanner.md

```diff
@@ -0,0 +1,70 @@
+[//]: # (title: Standard input)
+
+> Java Scanner is a slow tool. Use it only when you need the specific functionalities it offers.
+> Otherwise, it's generally preferable to use Kotlin's `readln()` function to [read standard input](basic-syntax.md#read-from-the-standard-input).
+>
+{type="note"}
+
+In Kotlin, you can use Java Scanner to read data from standard input. Due to Kotlin's interoperability with Java libraries,
+it's possible to access Java Scanner from Kotlin code out of the box.
+
+To use Java's Scanner, you need to first import it by adding the following statement at the top of your source code file:
+
+```Kotlin
+import java.util.Scanner
+```
+
+Initialize the scanner by creating a `System.in` object that represents the standard input stream and dictates how to read the data.
```

Comment:
The `System.in` is a static property and accessing it doesn't not create a new input stream.
Hence, should be "Initialize the scanner by **passing** a `System.in` object...".